http://bugs.gentoo.org/466034

851:24: warning: implicit declaration of function 'strncmp' [-Wimplicit-function-declaration]
853:32: warning: implicit declaration of function 'strlen' [-Wimplicit-function-declaration]
873:24: warning: implicit declaration of function 'strcmp' [-Wimplicit-function-declaration]
853:46: warning: incompatible implicit declaration of built-in function 'strlen' [enabled by default]
944:58: warning: incompatible implicit declaration of built-in function 'strlen' [enabled by default]

--- gdk-pixbuf/io-png.c
+++ gdk-pixbuf/io-png.c
@@ -26,6 +26,7 @@
 #include "config.h"
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h> /* strcmp, strlen, strncmp */
 #include <png.h>
 #include "gdk-pixbuf-private.h"
 
